platform/ios/src/MGLMapboxEvents.m

```diff
@@ -456,6 +456,8 @@ - (void) pushTurnstileEvent {
         
         // Flush
         [strongSelf flush];
```

Comment:
When telemetry is enabled, the turnstile will always be sent roughly simultaneously with the map load event. I wonder if we should reevaluate always flushing the event queue here.